src/main/java/org/craftercms/studio/api/v1/service/deployment/DeploymentService.java

```diff
@@ -73,36 +37,26 @@ List<ContentItemTO> getScheduledItems(String site, String sort, boolean ascendin
      *
      * @throws ServiceLayerException exception is case of en error
      */
-    void bulkGoLive(String site, String environment, String path, String comment) throws ServiceLayerException;
+    long bulkGoLive(String site, String environment, String path, String comment) throws ServiceLayerException, AuthenticationException;
 
     /**
      * Enable/Disable publishing for given site
-     * @param site site id
+     *
+     * @param site    site id
      * @param enabled true to enable publishing, false to disable publishing
      * @throws SiteNotFoundException
      */
-    boolean enablePublishing(String site, boolean enabled) throws SiteNotFoundException, AuthenticationException;
+    void enablePublishing(String site, boolean enabled) throws SiteNotFoundException, AuthenticationException;
 
     /**
      * Publish given commit IDs on given environment for given site
      * @param site site id to use for publishing
      * @param environment environment to use for publishing
      * @param commitIds commit IDs to publish
```

Comment:
What's the returned `long` for? Can we update the JavaDoc?

Reply:
Added to the javadoc. The returned long is the id of the create publish package